client/my-sites/checkout/checkout-thank-you/domains/thank-you-content/domain-registration.tsx

```diff
@@ -73,11 +88,13 @@ const DomainRegistrationThankYouProps = ( {
 			{
 				sectionKey: 'domain_registration_whats_next',
 				sectionTitle: translate( 'What’s next?' ),
-				nextSteps: [
-					...( professionalEmail ? [ professionalEmail ] : [] ),
-					...( ! selectedSiteSlug ? [ createSiteStep ] : [] ),
-					viewDomainsStep,
-				],
+				nextSteps: launchpadNextSteps
```

Comment:
`NB:` Could we avoid inlining ternaries in objects? We could use a temp variable instead.

```
const nextSteps = launchpadNextSteps
  ? [ launchpadNextSteps ]
  : [...originalNextSteps]
```